llvm/test/Transforms/Inline/promote-call-bitwidth.ll

```diff
@@ -0,0 +1,46 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --scrub-attributes
```

Comment:
```suggestion
; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --function-signature --scrub-attributes
;; Check that we correctly use the index size when accumulating offsets during CallPromotion
```
I had a comment on what the test is checking in https://github.com/CTSRD-CHERI/llvm-project/commit/fb9fa3765325776d07c547b1b0668e6c749120f3, IMO it's nice to add this to tests so it's obvious what is being tested without having to dig through the git history. I've spent way too much time in the past figuring out whether some downstream test diff was actually a problem or not and having comments like this makes it a bit easier.

Reply:
I had removed it because my impression is that such comments are not standard practice in upstream, and the specific assertion line may be brittle. However, if you feel strongly about it, I can add it back.

Reply:
I agree the comment I added was too specific - I suggested something slightly more generic.

https://llvm.org/docs/TestingGuide.html#best-practices-for-regression-tests says
> Include comments about what is tested/expected in a particular test. If there are relevant issues in the bug tracker, add references to those bug reports (for example, “See PR999 for more details”).

Reply:
Sorry, I missed that you'd provided a suggested comment here. Will incorporate.

Reply:
Done